src/lib.rs

```diff
@@ -109,7 +109,7 @@
 //!
 //! enr.set_tcp4(8001, &key);
 //! // set a custom key
-//! enr.insert("custom_key", &vec![0,0,1], &key);
+//! enr.insert("custom_key", &vec![0,0,1].as_slice(), &key);
```

Comment:
```suggestion
//! enr.insert("custom_key", &[0,0,1], &key);
```
These are just doctests so it's more of a style thing